schema/workerschema/v1.json

```diff
@@ -235,6 +254,24 @@
           "response": {
             "$ref": "ClientWithSecret"
           }
+        },
+        "Revoke": {
+          "id": "dex.Client.Revoke",
+          "description": "Revoke all refresh tokens issues to the client for the authenticated user.",
+          "httpMethod": "DELETE",
+          "path": "account/refresh/{userid}/clients/{clientid}",
+          "parameters": {
```

Comment:
shouldn't these be account/{userId}/refresh (swap positions of refresh and userID)?